packages/application-generic/src/services/bull-mq.service.ts

```diff
@@ -62,7 +81,19 @@ export class BullMqService {
     };
   }
 
-  public createQueue(name: string, config: QueueOptions) {
+  public createQueue(name: string, queueOptions: QueueOptions) {
+    const config = {
```

Comment:
Here and in `createWorker` we will do the discrimination of using either the Redis instance or the MemoryDB (in the next tasks). Yet think how to do it carefully as we will need to have the worker pulling from the old instance and in the new one for our cloud solution.